Lib/fontTools/ttLib/__init__.py

```diff
@@ -324,7 +329,8 @@ def saveXML(self, fileOrPath, progress=None, quiet=None,
 		if not hasattr(fileOrPath, "write") and fileOrPath != "-":
 			writer.close()
 
-	def _tableToXML(self, writer, tag, progress, quiet=None):
+	def _tableToXML(self, writer, tag, progress, tablePath, ttLibVersion=None, 
```

Comment:
i think tablePath should be an optional keyword argument `tablePath=None`, then below you assert it's not None when tag is "glyf"

Reply:
scrap that, we can do without passing tablePath and the other extra arguments for the writer, see my other comment https://github.com/fonttools/fonttools/pull/1035#discussion_r149935588